frontend/packages/console-dynamic-plugin-sdk/src/app/core/reducers/core.ts

```diff
@@ -41,6 +40,20 @@ export const coreReducer = (state: CoreState = { user: {} }, action: CoreAction)
         user: action.payload.userInfo,
       };
 
+    case ActionType.SetAdmissionWebhookWarning:
+      return {
+        ...state,
+        admissionWebhookWarning: {
+          warning: action.payload.warning,
+          kind: action.payload.kind,
+          name: action.payload.name,
+        },
```

Comment:
I think there is still a possibility for a race condition here. We need to be able to store more than one warning at a time.